### DISCUSSION TITLE
How is trailingSlash 'never' supposed to work on Vercel when static prerendered?

### DISCUSSION THREAD

**Discussion by alexbjorlig** [sveltejs/kit#10931] [Q&A]
I'm trying to understand the documentation [here](https://kit.svelte.dev/docs/page-options#trailingslash).

If we have a route `/docs/` deployed with static pre-render on Vercel, should it not redirect to `/docs`?

My coworker insists that we should update a property in `vercel.json` - but should it nok work without updating the `vercel.json` file?

**Comment by teemingc** [comment]
If you’re using adapter-vercel, I think Kit writes a static redirects file for you.

EDIT: same thing happens for adapter static
https://github.com/sveltejs/kit/blob/799b9e7cbf29e0b428b0be1d44c6a2631cbe8788/packages/adapter-vercel/index.js#L372
It writes to a config.json file
https://vercel.com/docs/build-output-api/v3/configuration#routes
